decidim-proposals/app/views/decidim/proposals/proposals/new.html.erb

```diff
@@ -1,3 +1,7 @@
+<% if translated_attribute(feature_settings.new_proposal_help_text).present? %>
```

Comment:
Sorry but I think the proposal text should appear just below the `New proposal` title 🌮 (here's a taco for you not to get mad)